lib/gruf/integrations/rails/railtie.rb

```diff
@@ -3,7 +3,7 @@
 module Gruf
   module Integrations
     module Rails
-      class Railtie < ::Rails::Railtie
+      class Railtie < Rails::Railtie
```

Comment:
This actually causes a fatal error:

> NameError: uninitialized constant Gruf::Integrations::Rails::Railtie

I'd prefer we keep the `::` prefixes across the system if possible, thanks.

Reply:
Yah, I understand, I brought all global scope resolution back.